### PR TITLE
Add services page

### PR DESCRIPTION
**Pull request by duranmla**
# Context
* Add basic layout and styles for services page
* Add illustrate headings code
* Add base for decorative background
* Finish responsiveness for services page
* Add specific SVG images for mobile screen

## Media
**Desktop**

hash_labs___hassle_free_software_engineering

**Tablet**
http://recordit.co/NFyoi9TnEH

**Mobile**
http://recordit.co/Bhyz3nJYKB

### CODE REVIEW

File: _sass/_objects--type.scss [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+.h2-responsive {
+  font-size: $font-size-base*1.75;
+
+  @include media-breakpoint-up(md) {
+    font-size: $font-size-h2;
+  };
+}
+
+.illustrated-heading {
```
Comment (TheMightyPenguin) [inline _sass/_objects--type.scss:40]: Why is this part of objects--type?

**Review by TheMightyPenguin — COMMENTED** [review]
Looks good, just a question before merge

File: _sass/_objects--type.scss (hunk 1 above)
Comment (duranmla) [reply _sass/_objects--type.scss:40]: Well, it is a heading, so it is a typography related style.

Comment (TheMightyPenguin) [reply _sass/_objects--type.scss:40]: It makes sense, could you put a comment above this class just to make clear where this is used?

**Review by TheMightyPenguin — APPROVED** [review]
LGTM

**Pull request merged**